orm/entities/SubmissionVersion/TallySheetVersion/TallySheetVersionPRE21.py

```diff
@@ -86,9 +89,16 @@ def html(self):
             if row.count is not None:
                 content["totalRejected"] = content["totalRejected"] + row.count
 
+        # convert image to data-uri
+        data_image = ''
+
+        with open("static/Emblem_of_Sri_Lanka.png", "rb") as img_file:
```

Comment:
have a util method to get this done since this is being used in many places

Reply:
updated in https://github.com/ECLK/results-tabulation-api/pull/348